src/js/application.js

```diff
@@ -0,0 +1,288 @@
+import { selectors } from './constants/selectors';
+import { classes } from './constants/classes';
+import { Location } from './location';
+import { Weather } from './weather';
+import { weatherIcons } from './weather-icons';
+import { translation } from './translation';
+
+export class Application {
+    constructor() {
+        this.degreeUnit = localStorage.getItem('degreeUnit') || 'celsius';
```

Comment:
optional: will be cool to move local storage keys to constants too

Reply:
fixed